.github/workflows/compaund-java-sonatype-push.yml

```diff
@@ -69,7 +69,7 @@ jobs:
           ORG_GRADLE_PROJECT_signingKey: ${{ secrets.sonatypeSigningKey }}
           ORG_GRADLE_PROJECT_signingPassword: ${{ secrets.sonatypeSigningPassword }}
       - name: Dev Build with Gradle
-        if: ${{ !inputs.closeAndRelease }}
+        if: !inputs.closeAndRelease && github.actor != 'dependabot[bot]'
```

Comment:
question: So, now we just don't build the component at all if it is a dependabot commit? Does not really make sense to me... If we want to skip publication we can do that like this

```
./gradlew -p ${{ inputs.projectPath }} --info clean build -Prelease_version=${{ format('{0}-SNAPSHOT', inputs.version) }} ${{ github.actor != 'dependabot[bot]' && 'publish' || '' }}
```